### ISSUE TITLE
Error while uploading Google drive

### ISSUE THREAD

**Issue by sumitaroy** [tshrinivasan/OCR4wikisource#70]
Creating a folder in Google Drive to upload files. Folder Name : OCR-Testocrbengali.djvu-temp-2016-02-18-01-32-46

INFO:**main**:Running gdmkdir.py "OCR-Testocrbengali.djvu-temp-2016-02-18-01-32-46" | tee folder_in_google_drive.log
Traceback (most recent call last):
  File "/usr/local/bin/gdmkdir.py", line 4, in <module>
    import pkg_resources
  File "/usr/lib/python2.7/dist-packages/pkg_resources.py", line 2749, in <module>
    working_set = WorkingSet._build_master()
  File "/usr/lib/python2.7/dist-packages/pkg_resources.py", line 446, in _build_master
    return cls._build_from_requirements(__requires__)
  File "/usr/lib/python2.7/dist-packages/pkg_resources.py", line 459, in _build_from_requirements
    dists = ws.resolve(reqs, Environment())
  File "/usr/lib/python2.7/dist-packages/pkg_resources.py", line 632, in resolve
    raise VersionConflict(dist,req) # XXX put more info here
pkg_resources.VersionConflict: (httplib2 0.8 (/usr/lib/python2.7/dist-packages), Requirement.parse('httplib2>=0.9.1'))
INFO:__main__:

uploading page_00001.pdf to google Drive. 
Traceback (most recent call last):
  File "do_ocr.py", line 295, in <module>
    command = "gdput.py -t ocr -f   " +  drive_folder_id + " "  + pdf_file + " | tee " + pdf_file.split('.')[0] + ".log"
NameError: name 'drive_folder_id' is not defined
sumita@sumita-System-Product-Name:~/Documents/OCR4wikisource-master$

**Comment by tshrinivasan** [comment]
What is the file_url?

Is this happening only for that file or other files too?

**Comment by sumitaroy** [comment]
Sorry for late reply. it was my first test file. After posting to that
forum I had searched for solution in net and seen some of your
solutions.According to that  I had reinstalled gdcmdtools-master .then
error was httplib version error I updated with httplib2-0.9.1 and then run
the program again then some security error was there. then I run the main
program with sudo and program run successfully .
    Thanks for this awesome program.Can you please make a flow chart type
of 'problem and probable solution' type of things for this prog with your
experience of  this program til now and feedback from users.
     Since I am a new user of Linux (just an week) and people like me will
be benefited from that maint. guide for this amazing program.
Regards.
Sumita
